main.go

```diff
@@ -52,8 +57,13 @@ func main() {
 
 	case "put":
 		key := flag.Arg(2)
-		value := valueOrReadFrom(flag.Arg(3), os.Stdin)
-		commandPutPasteGenerate(kmsService, storageService, target, "put", key, value)
+		value := flag.Arg(3)
+		if len(value) != 0 {
+			commandPutPasteGenerate(kmsService, storageService, target, "put", key, value, doPrompt)
+		} else {
+			value = readFromStdIn()
```

Comment:
Suggestion to put the usage of stdin in the README and `--help`, if available

Reply:
@emicklei Just wondering about this suggestion: What about using a `-f` flag for stdin and/or file like in `kubectl`? I think that's quite elegant as a solution.

Search for stdin: https://kubernetes.io/docs/reference/kubectl/overview/

Reply:
+1 for the -f flag that read from file.  Would like to keep stdin as well (do not break existing usage)